Form-Control/index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>My form exercise</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="styles.css">
+  <meta name="description" content="Order your custom t-shirt by choosing size, color, and preferred delivery date. Sign in to personalize your selection and complete your purchase easily.">
+
+
+
+</head>
+<body>
+  <header>
+    <h1>Your Tee, Your Way: Choose & Go!</h1>
+  </header>
+  <main>
+  <form id="tshirt-order-form">
+  <!---------------------------------------------- name ----------------------------------------------------------->
+    <div>
+      <label for="name">Name:</label>
+      <input type="text" name="name" id="name" placeholder="Enter your name" required minlength="2" required> <br>
+    </div>
+
+  <!---------------------------------------------- email ---------------------------------------------------------->
+    <div>
+      <label for="email">email:</label>
+      <input id="email" type="email" required name="email" placeholder="Enter your email">
+    </div>
+
+  <!---------------------------------------------- password ------------------------------------------------------->
+  <div>
+    <label for="password" >Password:
+      <input type="password" name="password" placeholder="password" required> <br>
+    </label>
+  </div>
+  <!---------------------------------------------- Delivery date -------------------------------------------------->
+  <div>
+    <label for="delivey-date" >Delivery date:
+      <input id="delivey-date" type="date" name="date" placeholder="delivey-date" required> <br>
+    </label>
```

Comment:
You did a great Job !  How we can be sure that it could be until next 4 weeks? (Based on the criteria of the Form in [Readme](https://github.com/RihannaP/Module-User-Focused-Data/blob/main/Form-Controls/README.md)).

Reply:
@Pezhman-Azizi Since we cannot obtain the current date without using JavaScript, for this exercise let's assume the current date is 1/11/2024. So how would you modify the input element to constrain the date selected by the user is one of the date within the next 4 weeks of 1/11/2024?

Reply:
@RihannaP Thanks so much for your feedback. There are actually two approaches to tackle such issue. 

1. Javascript to get the current date ( this link can be helpful 👉 (https://stackoverflow.com/questions/1531093/how-do-i-get-the-current-date-in-javascript). It is more of a dynamic approach. 

2. Second one which is a more static way is to use `min` and `max` attributes in the bespoke HTML tag. It limits the browser's calendar to a fix date. 

Reason I was wondering about using JavaScript was that I thought the `current date` by JavaScript gives a more realistic condition based on what date the user orders the T-Shirts.

Reply:
@cjyuan Thanks for the feedback! I understand that using JavaScript to dynamically calculate the date range would be the most flexible approach. However, given the constraint of not using JavaScript, I've opted to use the HTML min and max attributes to statically set the date range to four weeks from November 11th, 2024.

While this approach is less dynamic and requires manual adjustment if the base date changes, it effectively limits the user's date selection to the specified range within the constraints of the project.